account/account_test.go

```diff
@@ -4,13 +4,13 @@ import (
 	"context"
 	"testing"
 
-	"fmt"
-
 	"strings"
 
 	"io/ioutil"
 	"net/http"
 
+	"path"
```

Comment:
Remove the newline above `path`.  All of the stdlib packages should be part of the same comment block.